simplesat/test_data/__init__.py

```diff
@@ -0,0 +1,7 @@
+import os.path
```

Comment:
You've already added these to the package_data in `setup.py`, though, so it should already "just work."

We're already using that method for the scenario yaml files *and* it claims to be robust against zipped eggs. I also found it pretty easy to use while setting up airspeed-velocity for benchmarks.

Having a second way to compute (sometimes correct) paths won't interfere with that, but using this requires that eggs be unzipped, doesn't it?